conn_go19_test.go

```diff
@@ -0,0 +1,45 @@
+// +build go1.9
+
+package clickhouse
+
+func (s *connSuite) TestExecGO19() {
+	testCases := []struct {
+		query  string
+		query2 string
+		args   []interface{}
+	}{
+		{
+			"INSERT INTO data (u64) VALUES (?)",
+			"",
+			[]interface{}{uint64(maxAllowedUInt64 - 1)},
+		},
+		{
+			"INSERT INTO data (u64) VALUES (?)",
+			"",
```

Comment:
Please add "SELECT FROM data WHERE u64=?" to second query to check DB.Query as well.
Also need to add test for Statement.